service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProviderTest.java

```diff
@@ -0,0 +1,175 @@
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, CaseManagementOrderEmailContentProvider.class})
+class CaseManagementOrderEmailContentProviderTest extends AbstractEmailContentProviderTest {
+
+    private static final Long CASE_ID = 12345L;
+
+    @Autowired
+    private CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @PostConstruct
+    void setField() {
+        ReflectionTestUtils.setField(caseManagementOrderEmailContentProvider, "uiBaseUrl", BASE_URL);
+    }
+
+    @Test
+    void shouldBuildCMOIssuedCaseLinkNotificationParameters() {
+        Map<String, Object> expectedMap = ImmutableMap.<String, Object>builder()
+            .put("localAuthorityNameOrRepresentativeFullName", "testName")
+            .put("caseUrl", buildCaseUrl("12345"))
+            .put("subjectLineWithHearingDate", "lastName, 11")
+            .put("reference", "12345")
+            .build();
+
+        assertThat(caseManagementOrderEmailContentProvider
+            .buildCMOIssuedCaseLinkNotificationParameters(createCase(), "testName"))
+            .isEqualTo(expectedMap);
+    }
+
+    @Test
+    void shouldBuildCMOIssuedWithoutDocumentLinkNotificationParameters() {
+
+        Map<String, Object> expectedMap = ImmutableMap.<String, Object>builder()
+            .put("cafcassOrRespondentName", "testName")
+            .put("caseUrl", buildCaseUrl("12345"))
+            .put("subjectLineWithHearingDate", "lastName, 11")
+            .put("reference", "12345")
+            .build();
+
+        assertThat(caseManagementOrderEmailContentProvider.buildCMOIssuedDocumentLinkNotificationParameters(
+            createCase(), "testName", new byte[]{}))
+            .isEqualTo(expectedMap);
+    }
+
+    @Test
+    void shouldBuildCMOIssuedWithDocumentLinkNotificationParameters() throws JsonProcessingException {
+
+        byte[] documentContentAsByte = nextBytes(20);
+        String documentContent = new String(Base64.encodeBase64(documentContentAsByte), ISO_8859_1);
+
+        JSONObject expectedDocumentLink = new JSONObject().put("file", documentContent);
+
+        Map<String, Object> expectedMap = ImmutableMap.<String, Object>builder()
+            .put("cafcassOrRespondentName", "testName")
+            .put("caseUrl", buildCaseUrl("12345"))
+            .put("subjectLineWithHearingDate", "lastName, 11")
+            .put("link_to_document", expectedDocumentLink)
+            .put("reference", "12345")
+            .build();
+
+        String expected = objectMapper.writeValueAsString(expectedMap);
+        String actual = objectMapper.writeValueAsString(
+            caseManagementOrderEmailContentProvider.buildCMOIssuedDocumentLinkNotificationParameters(
+                createCase(), "testName", documentContentAsByte));
+
+        JSONAssert.assertEquals(expected, actual, true);
```

Comment:
Map<String, Object> actualParameters =
            caseManagementOrderEmailContentProvider.buildCMOIssuedDocumentLinkNotificationParameters(
                createCase(), "testName", documentContentAsByte);

        AssertionHelper.assertEquals(actualParameters, expectedParameters);

then remove ObjectMapper and throws JsonProcessingException